pkg/compose/up.go

```diff
@@ -117,15 +122,27 @@ func (s *composeService) Up(ctx context.Context, project *types.Project, options
 			case <-signalChan:
 				if first {
 					gracefulTeardown()
-				} else {
-					eg.Go(func() error {
-						return s.Kill(context.Background(), project.Name, api.KillOptions{
-							Services: options.Create.Services,
-							Project:  project,
-						})
-					})
-					return nil
```

Comment:
I am removing this return, so it is possible now to try to kill the containers again after one kill failed.

Reply:
This should not be necessary, I hardly can imagine process could survive a SIGKILL
Also I'm not sure about the potential side-effects doing so, so if you don't mind I'd prefer to be conservative here.

Reply:
Well, indeed hopefully no process is going to survive a SIGKILL :slightly_smiling_face: But this is sending requests to kill containers to the docker daemon through HTTP, so there are other things that could fail in the way.

In any case I agree with being conservative and keep previous behaviour, I will revert this.